src/main/scala/akka/persistence/jdbc/migraition/V4Migration.scala

```diff
@@ -38,32 +39,47 @@ class V4JournalMigration(config: Config, system: ActorSystem) extends JournalTab
   def run(): Unit = {
     val db = SlickDatabase.forConfig(config, journalConfig.slickConfiguration)
     try {
-      println(s"Migrating journal events, each . indicates $RowsPerTransaction rows migrated.")
-      val migration = migrateNextBatch(db, 0)
-      val migrated = Await.result(migration, Duration.Inf)
-      println(s"Journal migration complete! $migrated events were migrated.")
+      val migration = for {
+        eventsToMigrate <- countEventsToMigrate(db)
+        _ = log.info(s"Migrating $eventsToMigrate journal events in batches of $RowsPerTransaction.")
+        migrated <- migrateNextBatch(db, migrated = 0, orderingFrom = 0L)
+      } yield {
+        log.info(s"Journal migration complete! $migrated events were migrated.")
```

Comment:
For all log statements, the format syntax `log.info("Journal migration complete! {} events were migrated.", migrated)` should be preferred, since this saves some string operations in case the logger is disabled